### DISCUSSION TITLE
Remove "missing" snapshot from Percy project

### DISCUSSION THREAD

**Discussion by digio-peterbaldwin** [percy/cli#653] [Support] [answered]
During my investigation of percy.io, the percy project's configured screenshots have altered. I would now like to ignore two screenshots (which were included in my percy project's original configuration).

**Comment by Robdel12** [comment, marked as answer]
Hey @digio-peterbaldwin! I'm not sure I understand what you're asking. It kind of sounds like you want to update your baseline? If so, you'll want to run another build on the baseline branch (`master` by default). These docs will also be helpful: https://docs.percy.io/docs/baseline-picking-logic
